lib/simulator-xcode-9.js

```diff
@@ -12,6 +12,15 @@ const SIMULATOR_SHUTDOWN_TIMEOUT = 15 * 1000;
 const startupLock = new AsyncLock();
 const preferencesPlistGuard = new AsyncLock();
 
+const BIOMETRICS = {
```

Comment:
I expect this structure to be exported by this lib. You could make it a static class field